examples/python/RVC2/ToF/tof_minimal.py

```
@@ -1,64 +1,51 @@
+        min_depth = np.percentile(frame_depth[frame_depth != 0], 3)
+        max_depth = np.percentile(frame_depth[frame_depth != 0], 95)
+        log_depth = np.log(frame_depth, where=frame_depth != 0)
+        log_min_depth = np.log(min_depth)
+        log_max_depth = np.log(max_depth)
+        np.nan_to_num(log_depth, copy=False, nan=log_min_depth)
+        log_depth = np.clip(log_depth, log_min_depth, log_max_depth)
+        depth_color = np.interp(log_depth, (log_min_depth, log_max_depth), (0, 255))
+        depth_color = depth_color.astype(np.uint8)
+        depth_color = cv2.applyColorMap(depth_color, cv2.COLORMAP_JET)
+        depth_color[invalid_mask] = 0
+        return depth_color
```

Ruff (0.15.15)

[warning] Consider moving this statement to an `else` block

(TRY300)